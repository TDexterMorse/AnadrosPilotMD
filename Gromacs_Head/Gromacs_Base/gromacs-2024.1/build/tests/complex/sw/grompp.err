                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/complex/sw
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/sw/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/sw/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/sw/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/sw/topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize_fft'
Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'ns_type'

NOTE 1 [file topol.top, line 162]:
  In moleculetype 'SW' 1 atoms are not bound by a potential or constraint
  to any other atom in the same moleculetype. Although technically this
  might not cause issues in a simulation, this often means that the user
  forgot to add a bond/potential/constraint or put multiple molecules in
  the same moleculetype definition by mistake. Run with -v to get
  information for each atom.

Number of degrees of freedom in T-Coupling group System is 1509.00

There was 1 NOTE

Thanx for Using GROMACS - Have a Nice Day

