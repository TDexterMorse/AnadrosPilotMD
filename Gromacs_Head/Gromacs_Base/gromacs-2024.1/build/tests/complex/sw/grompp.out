Generated 6 of the 10 non-bonded parameter combinations

Excluding 2 bonded neighbours molecule type 'SW'

turning all bonds into constraints...

Cleaning up constraints and constant bonded interactions with virtual sites
Analysing residue names:
There are:   216      Other residues
Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...

The largest distance between excluded atoms is 0.164 nm between atom 682 and 683

Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 298.15 K

Calculated rlist for 1x1 atom pair-list as 0.840 nm, buffer size 0.040 nm

Set rlist, assuming 4x4 atom pair-list, to 0.804 nm, buffer size 0.004 nm

Note that mdrun will redetermine rlist based on the actual pair-list setup
Using a fourier grid of 32x32x32, spacing 0.058 0.058 0.058

Estimate for the relative computational load of the PME mesh part: 0.40

This run will generate roughly 0 Mb of data
